cms-YYYY-validated-runs/code/vali_records.py

```diff
@@ -31,11 +58,26 @@ def create_record(recid, year, filename):
         muon_text = ''
         muon_desc = ''
 
-    
+    run_range_input = era
+    if "pphiref" in runtype:
+        collision_text = energy+' proton-proton collision data, needed as reference data for heavy-ion data analysis,'
```

Comment:
If we can assume Python 3 being used, then we could reformulate the expression to use f-strings, for example:

```python
        collision_text = f'{energy} proton-proton collision data, needed as reference data for heavy-ion data analysis,'
```

which might be more readable?